dependent.js

```diff
@@ -57,10 +57,14 @@ class Dependent {
     //
     if (dependent) {
       return ReleaseLineDep.remove({ pkg, version, dependent });
+    } else {
+      let dependencyRemoves = [];
+      const dependents = await ReleaseLineDep.findAll({ pkg, version })
```

Comment:
Can you double check that eslint is running and catching all the files.
```suggestion
      const dependents = await ReleaseLineDep.findAll({ pkg, version });
```